spikes/Firewall-Automation/Firewall-Automation-Manual-Installation.md

```diff
@@ -0,0 +1,427 @@
+|  3. | User-Assigned Managed Identity            | [link](https://learn.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/how-manage-user-assigned-managed-identities?pivots=identity-mi-methods-azcli) | Create an identity for the Automation Account.                                                                                                                              |   |
+|  4. | Automation Runbook with Powershell        |                                      [link](https://learn.microsoft.com/en-us/azure/automation/automation-runbook-types#powershell-runbooks)                                     | Create a Runbook of type Powershell.                                                                                                                                        |   |
+|  5. | Powershell Content in Runbook             |                               [link](https://learn.microsoft.com/en-us/powershell/module/az.automation/import-azautomationrunbook?view=azps-8.3.0)                               | Upload [pre-defined Powershell content](../automation/FirewallToggle.ps1) into the Runbook body.                                                                            |   |
+|  6. | Automation Schedule(s) _using Powershell_ |                               [link](https://learn.microsoft.com/en-us/powershell/module/az.automation/import-azautomationrunbook?view=azps-8.3.0)                               | Create the schedules that will execute the Firewall automation.  These had to be created using Powershell instead of the Azure CLI.  No equivalent behavior has been found. |   |
+
+### Resources Created When Complete
+
+1.  Azure user-assigned Managed Identity
+2.  Azure Automation Account
+3.  Azure PowerShell Runbook
+
+    ![List of resources that will be created when complete.](./assets/automation/listOfResourcesInResourceGroup.png)
+
+## Choosing the Installation Method
+
+The instructions below show three manners to accomplish the same objective.  
+
+Choose **ONLY** one.
+
+1.  [Portal](#portal) - Using the Azure Portal to create all artifacts
+2.  [CLI & Az PowerShell Modules](#azure-cli-and-az-powershell-modules) - Using a terminal and executing the PowerShell and Bash CLI commands interactively
+3.  Executing the Bash and PowerShell Scripts Provided.  
+
+### Portal
+
+Note if you chose a "portal based installation" please follow the instructions below.  
+
+Create a resource group within the subscription and tenant that contains the firewalls.  This new resource group will house three resources as shown below.
+
+![Create a new Resource Group](./assets/automation/createResourceGroup1Portal.png)
+
+#### Create User-Assigned Managed Identity
+
+Navigate to the newly created resource group and click the "+" to add a new resource
+
+Search for "User Assigned Managed Identity".
+
+![Create a User-Assigned Managed Identity for the Automation](./assets/automation/createUserAssginedManagedIdentityPortal.png)
```

Comment:
Please update broke links. ./assets/automation/createUserAssginedManagedIdentityPortal.png should point to ./assets/Firewall-Automation/createUserAssginedManagedIdentityPortal.png

Reply:
I deleted the "spike" folder for Firewall-Automation as it wasn't really a spike but different ways to do the same task; more technical debt in my opinion.